docs/generated/mass_image_push.md

```diff
@@ -12,7 +12,7 @@ Push an image to ECR, ACR or GAR
 
 # Push Container Images To Cloud Repositories
 
-Create registries, repositories and push images via the Massdriver CLI. Massdriver will build a Docker registry if it does not exist in the region in which you are pushing an image, create a repository in that region's registry and finally push a tagged version of the image to that repository.
+Create registries, repositories and push images via the Massdriver CLI. Massdriver will build a Docker registry if it does not exist in the region in which you are pushing an image, create a repository in that region's registry and finally push a tagged version of the image to that repository. By default `mass` will attempt to build your image before pushing it. To disable this feature pass the `--skip-build` flag.
```

Comment:
should we add some instructions about how to reference a prebuilt image?

Reply:
Added an example below